docs/FAQ/gui-faq-en.md

```diff
@@ -20,147 +20,148 @@ Zonemaster
-to. It is quite rare that you'd want a complete program just to check for example
+which basically means that you can integrate parts of it in your own systems, if you wish.
+For example, it is quite rare that you would want a complete program just to check for
 redelegations.
 
 <a name="q9"></a>
 #### 9. Zonemaster and privacy
 Since Zonemaster is open to everyone it is possible for anyone to check your
-domain and also see previous tests, however there is no way to tell
-who has run a specific test since nothing is logged except the time of the test.
+domain and its history of tests.
+However there is no way to tell who has run a specific test since nothing is
+logged except the start time of the test.
 
 <a name="q10"></a>
-#### 10. How come I cannot test my domain?
-If we disregard the situation where the domain does not exist, as in when you input a
-non-existing domain to Zonemaster, there are 2 other possibilites:
-  - To protect the engine from multiple identical inputs, that is the same IP
-    checking the same zone several times, there is a delay of 5 minutes between
-    identical subsequent tests. This means that you can only test the
-    same domain once every 5 minutes, if you try and test it again within 5 minutes
-    the last results will be displayed instead.
-  - Because Zonemaster was made to check domains or DNS zones, like zonemaster.net, and
-    not hostnames in a domain (zone), like www.zonemaster.net, the Zonemaster will
-    report a failure if you try to test a host name instead of a domain matching a
-    DNS zone.
+#### 10. How come my domain cannot be tested?
+There are four possibilities:
+
+- Your domain name is not yet delegated.
+- Your domain name is not reachable from public Internet.
+- Zonemaster can only test what is called a DNS zone (e.g. 'zonemaster.net') and not host names (e.g. 'www.zonemaster.net')
+- There is a 10 minutes protection, per IP address, between consecutive tests for a given domain name.
+  Running a test within that window will instead show the last available test for that domain name.
+- You have misspelled your domain name.
 
 <a name="q11"></a>
 #### 11. What kind of queries does Zonemaster generate?
-Zonemaster send multiple DNS queries to the name servers hosting the domain name and
-also to the name servers hosting the parent zone of the domain name.
+Zonemaster send multiple DNS queries to the name servers hosting the domain being tested and
+also to the name servers hosting the parent zone of that domain.
 
-To get a full
-view of what queries and results are generated you can run the
-CLI interface (and in order to run the CLI interface you need to download the
-complete package and install it) and select full output.
-The output from the CLI tool is quite heavily technical
-so unless you are into bits and bytes you might want to skip this step.
+The GUI interface of Zonemaster does not show any queries sent.
```

Comment:
Isn't GUI interface and CLI interface redundant? Maybe use the extended wording `graphical interface of Zonemaster` and `command line interface of Zonemaster` or drop the extra `interface`.

Reply:
Technically, "interface" is redundant, but this a very common way to make it easy to read. Compare with "DNS system" and "RCODE codes". I think "GUI interface" is fine. It is more compact than "graphical user interface".

Reply:
Unchanged. I also like the redundancy here.